tokens/eth/0xE62bE16aa3c50baFb1855b4dEd7799D9150BB1A1.json

```diff
@@ -0,0 +1,34 @@
+{
+  "symbol": "DYL",
+  "address": "0xE62bE16aa3c50baFb1855b4dEd7799D9150BB1A1",
+  "decimals": 18,
```

Comment:
"Amount"

Reply:
void token I believe this client was hacked through the thirdweb exploit.

Reply:
Wtf

Reply:
> 19